parallel/example/net_with_flow_info.py

```diff
@@ -2,7 +2,7 @@
 from time import time
 import pandas as pd
 from mpi4py import MPI
-from json import load, dump
+from json import dump
```

Comment:
why not change code at [here](https://github.com/sequence-toolbox/SeQUeNCe/blob/f8c326e6aebb98f92f5de60e345b0f5da9463982/parallel/example/net_with_flow_info.py#L33)?